scripts/ts-utils/setup-enclave-script.ts

```diff
@@ -0,0 +1,38 @@
+
+//run:npx ts-node setup-enclave-script.ts
+
+import { initApi } from "./index";
+const { hexToU8a } = require('@polkadot/util');
+
+//100 token
+const transferAmount = '100000000000000';
+const enclaveAccount = '5EtChN6SSfL7E13fEqKQn4jZztDtZkSiRqfFTv2BKufL5fzZ'
+const mrenclave = 'a552654d1733c4054a3c7e5e86adf26b5d65c072b57b2550fe763821ebac54c6'
```

Comment:
Please get these 3 variables from the script's command line args.

Reply:
The same I want to comment.